pom.xml

```diff
@@ -28,7 +28,7 @@
         <releases.repository.url />
         <admin-query.version>1.2.7</admin-query.version>
```

Comment:
Are we also going to need to bump the admin-query version?

Reply:
I think it will pull in a version range of DDF. Having trouble checking right now though cause solr is broken, but will check before merging.

Reply:
Confirmed no 2.13.6 DDF bundles are started from admin-query.